src/components/ThumbnailsPanel/ThumbnailsPanel.js

```diff
@@ -271,6 +378,8 @@ class ThumbnailsPanel extends React.PureComponent {
         className="Panel ThumbnailsPanel"
         style={{ display }}
         data-element="thumbnailsPanel"
+        ref={this.thumbnailsPanel}
+        onDragOver={this.dragOverHandler}
```

Comment:
I think we can just use `e => e.preventDefault` here instead of having a function. What do you think? We do similar things in other components.

Reply:
We can do that but I normally avoid creating anonymous function in the render but if we are doing it in other places I can change this to just

```
onDragOver={e => e.preventDefault()}
```

Reply:
I actually ended getting rid of this, in the "Thumbnail" "onDragOver" callback, I added 
`e.stopPropagation();` so it never get throw dragOver event for the parent.